### DISCUSSION TITLE
Get all unique values of a given columnId

### DISCUSSION THREAD

**Discussion by xinli-wu** [TanStack/table#4984] [Ideas]
Can we have a function (or 2 functions) to get all unique values (unfaceted/unfiltered) of a column? They could be:

1. column.getAllUniqueValues(), and/or
2. table.getAllUniqueValues(columnId)

Thanks!

**Comment by KevinVandy** [comment]
We already do. getFacetedUniqueValues

**Reply by xinli-wu** [reply]
if I have column filters applied, getFacetedUniqueValues will only give me filtered unique values, not all unique values. See this https://github.com/TanStack/table/discussions/4981
